### ISSUE TITLE
Conduit dashboard with --url option shouldn't care if there's another proxy running

### ISSUE THREAD

**Issue by pcalcado** [runconduit/conduit#333]
Steps:
1. Run `kubectl proxy` in a terminal
2. Run `conduit dashboard --url` in another terminal

What did I expect:
Conduit returns a url I can copy and paste

What happens:
```shell
pcalcado@arbeitsziege.local $ conduit dashboard Failed to initialize proxy: Failed to listen with proxy: Failed to listen via proxy server: listen tcp 127.0.0.1:8001: bind: address already in use
```

**Comment by briansmith** [comment]
Did @klingerf's recent changes to use a random port fix this?

**Comment by klingerf** [comment]
Am going to close this as fixed by #394.
